modules/abstract-utxo/src/abstractUtxoCoin.ts

```diff
@@ -516,19 +516,19 @@ export abstract class AbstractUtxoCoin extends BaseCoin {
       txInfo: txPrebuild.txInfo,
       pubs: keychainArray.map((k) => k.pub) as Triple<string>,
     });
-
     const allOutputs = [...explanation.outputs, ...explanation.changeOutputs];
 
     let expectedOutputs;
     if (txParams.rbfTxIds) {
       assert(txParams.rbfTxIds.length === 1);
-      const txToBeReplacedTransfer = await wallet.getTransfer({ id: txParams.rbfTxIds[0] });
+
+      const txToBeReplaced = await wallet.getTransaction({ txHash: txParams.rbfTxIds[0], includeRbf: true });
       // Note: Will work only when there is single transaction output per address
       // TODO: https://bitgoinc.atlassian.net/browse/BTC-826
```

Comment:
```suggestion

```

Reply:
Oh yeah using transaction doesn't warrant this TODO now.

Reply:
Raising PR.